### PR TITLE
Feature/ainesh01/organization schema

### LINKED ISSUE #21: Backend - Create organizations schema
- Create organizations schema in the models folder with the following fields
- Name (string, required, index true)
- Email (string, required)
- Password (string, required)
- Approved Users (array of objectIds, reference users collection, not required)
- Status (string, required, default: "pending")
- Timestamps true

[Getting started with references](https://harrisonstandeffer.com/simulating-sql-joins-with-mongoose/)

### PR DESCRIPTION
**Pull request by navidboloorian**
### Administrative Info
[GitHub Issue #] closing issue 21
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
Added Organizations Schema

### Testing
How did you confirm your changes worked? 
Ran testing code

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.

image

### CODE REVIEW

File: backend/models/Users.js [hunk 1]
```diff
@@ -55,11 +55,12 @@ UserSchema.pre("findOneAndUpdate", async function (next) {
   }
 });
 
-UserSchema.methods.verifyPassword = (enteredPassword) =>
+UserSchema.methods.verifyPassword = function (enteredPassword) {
```
Comment (eliasfang) [inline backend/models/Users.js:58]: Missing a function header I believe

**Review by eliasfang — CHANGES_REQUESTED** [review]
Nice job overall! Just a small change. Also for future PR's I'd recommend adding a bit more description with the changes you made and how you tested it so that we can try reproducing what you got.

File: backend/models/Organizations.js [hunk 2]
```diff
@@ -64,6 +70,11 @@ OrganizationSchema.pre("findOneAndUpdate", async function (next) {
   }
 });
 
+/**
+ * Verifies that entered password matches ecnrypted password in the db
+ * 
+ * @param {*} enteredPassword - entered password
+ */
 OrganizationSchema.methods.verifyPassword = function (enteredPassword) {
   bcrypt.compare(enteredPassword, this.password, (err, matchBool) => {
```
Comment (pratyush1718) [inline backend/models/Organizations.js:79]: Can we change this to a regular function instead of arrow? Just for consistency

**Review by pratyush1718 — APPROVED** [review]
LGTM

**Review by eliasfang — APPROVED** [review]
Don't forget to do lint checks and fix any conflicts with main and then you should be good to go!

**Pull request merged**